src/main/groovy/org/shipkit/gradle/notes/UpdateReleaseNotesTask.java

```diff
@@ -129,6 +129,13 @@ public String getGitHubRepository() {
     }
 
     /**
+     * @return a link to the generated release notes file hosted on Github.
+     */
+    public String getReleaseNotesUrl(String branch) {
+        return  getGitHubUrl() + "/" + getGitHubRepository() + "/blob/" + branch + "/" + getProject().relativePath(getReleaseNotesFile());
```

Comment:
Nice new property!

Let's push all the implementation to the "UpdateReleaseNotes" class. This way, the public API types have only a reason to change when public API changes!